private/buf/bufcli/bufcli.go

```diff
@@ -549,8 +549,7 @@ func NewConfig(container appflag.Container) (*bufapp.Config, error) {
 	return bufapp.NewConfig(container, externalConfig)
 }
 
-// NewRegistryProvider creates a new registryv1alpha1apiclient.Provider.
-func NewRegistryProvider(ctx context.Context, container appflag.Container) (registryv1alpha1apiclient.Provider, error) {
+func newGRPCRegistryProvider(ctx context.Context, container appflag.Container) (registryv1alpha1apiclient.Provider, error) {
```

Comment:
Public functions always go above private, ie move this down the file to below the public functions

Reply:
Got it.  Moved the private ones down in the file.